### PR TITLE
feat(schematics): switch to angular 5.2 and cli 1.6.5

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: packages/schematics/src/collection/application/files/__directory__/tslint.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
     "callable-types": true,
     "class-name": true,
     "forin": true,
+    "deprecation": {
+      "severity": "warn"
```
Comment (ThomasBurleson) [inline packages/schematics/src/collection/application/files/__directory__/tslint.json]: Why did you need this?

Comment (vsavkin) [reply packages/schematics/src/collection/application/files/__directory__/tslint.json]: This is to match what CLI 1.6.5 generates by default.
